crates/red_knot_module_resolver/src/lib.rs

```diff
@@ -9,10 +9,40 @@ mod typeshed;
 #[cfg(test)]
 mod testing;
 
+use crate::resolver::{module_resolution_settings, SearchPathIterator};
 pub use db::{Db, Jar};
 pub use module::{Module, ModuleKind};
 pub use module_name::ModuleName;
 pub use resolver::resolve_module;
+use ruff_db::system::SystemPath;
+use std::iter::FusedIterator;
 pub use typeshed::{
     vendored_typeshed_stubs, TypeshedVersionsParseError, TypeshedVersionsParseErrorKind,
 };
```

Comment:
I think we should either deploy tooling to standardize use sorting or not be opinionated about it. It otherwise is guaranteed to get out of sync.

Reply:
Fair enough, I'll try to be less picky about it in the future :(

Reply:
FWIW I'd be happy to have tooling that standardizes it, but I haven't looked into what the options are.

Reply:
Rustfmt has an option for it but it isn't stabilized. https://rust-lang.github.io/rustfmt/?version=master&search=import#group_imports and it can't be enabled on non-nigthly rustfmt

```
Warning: can't set `group_imports = StdExternalCrate`, unstable features are only available in nightly channel.
Warning: can't set `unstable_features = true`, unstable features are only available in nightly channel.
```